src/server/views/dashboard/macros.njk

```diff
@@ -50,101 +51,90 @@
 {% endmacro %}
 
 {% macro listItemEditActionBar(list, listItem, dashboardRoutes, titleProperty, isPinned, actionButtons, changeMessage) %}
+  {% set actionToLabelText = {
+      publish: "Publish",
+      pin: "Pin",
+      remove: "Remove",
+      requestChanges: "Request changes",
+      unpin: "Unpin",
+      unpublish: "Unpublish",
+      update: "Update",
+      updateLive: "Update",
+      updateNew: "Publish",
+      archive: "Archive"
+    }
+  %}
 
+  {% set changeMessageHtml %}
+    {{ govukTextarea ({
+      id: "message",
+      name: "message",
+      label: {
+        text: "Change message",
+        classes: "govuk-visually-hidden"
+      },
+      hint: {
+        text: "Provide simple and concise instructions to the service provider on what they need to change."
+      }
+    }) }}
+  {% endset %}
```

Comment:
IF there is an endset here and one on line 94, should there be one below line 66?

Reply:
Ah yes, I see that now 😉